### PR TITLE
Adding date time function for logs

### PR DESCRIPTION
**Pull request by bioinformike**
I added a function to add the date and time to any output messages, for improved tracking in the log file.

### CODE REVIEW

File: cloudflare-template.sh [hunk 1]
```diff
@@ -6,27 +6,36 @@ zone_identifier=""                                 # Can be found in the "Overvi
 record_name=""                                     # Which record you want to be synced
 proxy=true                                         # Set the proxy to true or false 
 
+
+###########################################
+## Define date time stamp function
+###########################################
+function ds() {
+            date +"[%F %T]"
```
Comment (K0p1-Git) [inline cloudflare-template.sh:14]: Hi, thanks for your contribution. Would it be possible to keep the white space consistent?

Comment (bioinformike) [reply cloudflare-template.sh:14]: Sorry, how do you mean?

Comment (bioinformike) [reply cloudflare-template.sh]: I think I understood. I changed the tab in the function definition to 2 spaces to match the indentation you were using in your if statements.

Comment (K0p1-Git) [reply cloudflare-template.sh]: The indentation on line 14, would it be possible to keep it at 2, like the rest of the script?

Comment (bioinformike) [reply cloudflare-template.sh]: Got it. I just committed a change that did this.

**Comment by K0p1-Git** [comment]
Lastly, have you tested the script?

**Comment by bioinformike** [comment]
Yep, it's working well for me.

> [2021-07-19 07:15:04] IP (0.0.0.0) for redacted.com has not changed.
[2021-07-19 07:20:01] Check Initiated
[2021-07-19 07:20:04] IP (0.0.0.0) for redacted.com has not changed.
[2021-07-19 07:25:02] Check Initiated
[2021-07-19 07:25:04] IP (0.0.0.0) for redacted.com has not changed.
[2021-07-19 07:30:01] Check Initiated
[2021-07-19 07:30:04] IP (0.0.0.0) for redacted.com has not changed.
[2021-07-19 07:32:19] Check Initiated
[2021-07-19 07:32:19] 0.0.0.0 redacted.com DDNS updated.

**Comment by K0p1-Git** [comment]
Looks good to me!

**Pull request merged**